src/data/ores/index.js

```diff
@@ -0,0 +1,5 @@
+export { bronzeOre } from './bronzeOre'
+export { goldOre } from './silverOre'
+export { ironOre } from './silverOre'
+export { silverOre } from './silverOre'
```

Comment:
It appears that these `import`s are incorrect and are breaking the build: https://github.com/jeremyckahn/farmhand/pull/159/checks?check_run_id=3087808129

Reply:
I've fixed this in https://github.com/jeremyckahn/farmhand/commit/e70347d2859bd571c4bb5fff482b9032ac47d04c!

Reply:
thank you! silly copypasta